.gitignore

```diff
@@ -50,6 +50,9 @@ coverage.xml
 .pytest_cache/
 cover/
 
+# Integration test 
+test_media_file.txt
```

Comment:
Should this file be in the repo?

Reply:
no, i have since changed the way we add the test file to the bucket. this can be removed now